### PR TITLE
This PR adds ESLint and Prettier configuration to improve code quality and formatting. 

### PR DESCRIPTION
**Pull request by GamilaCoding**
### Changes Made
- Added `.eslintrc.json` for ESLint configuration.
- Added `.prettierrc` for Prettier configuration.
- Updated `package.json` with linting scripts.

### CODE REVIEW

**Comment by SureshPradhana** [comment]
@GamilaCoding I made a review; could you please update the changes accordingly? Thank you for contributing!

File: package.json [hunk 1]
```diff
@@ -29,10 +30,17 @@
     "@tailwindcss/typography": "^0.5.14",
     "@types/eslint": "^9.6.0",
     "autoprefixer": "^10.4.20",
-    "eslint": "^9.0.0",
+    "eslint": "^9.13.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-prettier": "^5.2.1",
+    "eslint-plugin-react": "^7.37.1",
```
Comment (SureshPradhana) [inline package.json]: Please remove `eslint-plugin-react` since we are using Svelte.

File: yarn.lock
Comment (SureshPradhana) [inline yarn.lock:1]: Please use npm for consistency and delete the `yarn.lock` file.

**Review by SureshPradhana — COMMENTED** [review]
I have reviewed the changes and requested some modifications.

File: yarn.lock
Comment (GamilaCoding) [reply yarn.lock:1]: done

**Comment by GamilaCoding** [comment]
@SureshPradhana 
I've made the requested changes as per your feedback. Here’s what I’ve updated:

- remove yarn.lock file
-  remove plugin eslint react

Please review the updated code and let me know if further adjustments are needed. Thanks for your guidance!

**Pull request merged**